std/traits.d

```diff
@@ -8590,3 +8591,119 @@ enum isCopyable(S) = is(typeof(
     static assert(isCopyable!int);
     static assert(isCopyable!(int[]));
 }
+
+/++
+ + Determines if `F` is a strongly `pure` function. To be strongly `pure`, a function must satisfy the following:
```

Comment:
To prevent redundancies, i'd go for something like this instead:
`Determines if a function is string `pure`", which is the case when...`

Reply:
Done